### PR TITLE
gzip response if header is present 

### PR DESCRIPTION
**Pull request by kartikag01**

### CODE REVIEW

File: packages/serverless-nextjs-component/next-aws-cloudfront.js [hunk 1]
```diff
@@ -125,14 +126,32 @@ const handler = event => {
       Buffer.isBuffer(chunk) ? chunk : Buffer.from(chunk)
     ]);
   };
+  let gz = false;
+  const ae = headers["accept-encoding"];
```
Comment (barrysteyn) [inline packages/serverless-nextjs-component/next-aws-cloudfront.js]: Can this be refactored into it's own function?

Comment (kartikag01) [reply packages/serverless-nextjs-component/next-aws-cloudfront.js]: yes definitely

Comment (kartikag01) [reply packages/serverless-nextjs-component/next-aws-cloudfront.js]: Hey, @barrysteyn I have done required changes.

**Comment by kartikag01** [comment]
Hey @barrysteyn could you review updated PR.

**Comment by danielcondemarin** [comment]
@kartikag01 Could you add a test to ensure there are no regression issues introduced later on?

**Comment by danielcondemarin** [comment]
See a quick update on some more thoughts I have about GZIP [here](https://github.com/danielcondemarin/serverless-next.js/issues/201#issuecomment-578396884)

**Comment by danielcondemarin** [comment]
@kartikag01 Sorry for the PR taking so long to be merged! Great work and this is now available in version @1.9.4 🚀

**Pull request merged**